### PR TITLE
data: DhanushNehru

### PR DESCRIPTION
**Pull request by DhanushNehru**

### CODE REVIEW

File: data/DhanushNehru.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "date": "January 2023",
+      "icon": "FaHashnode",
+      "color": "blue",
+      "description": "Documented my journey as a developer in 2022",
+      "url": "https://dhanushnehru.hashnode.dev/dev-retro-2022-journey-in-review"
+    },
+    {
+      "title": "Completed 100DaysOfCode",
+      "date": "December 2020",
+      "icon": "FaHashnode",
```
Comment (Aadarsh805) [inline data/DhanushNehru.json:41]: Use `SiHashnode` for Hashnode icon, the `FaHashnode` doesn't work

File: data/DhanushNehru.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      "name": "Follow me on CodePen",
+      "url": "https://codepen.io/dhanushnehru",
+      "icon": "FaCodepen"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Documented dev journey of 2022",
+      "date": "January 2023",
+      "icon": "FaHashnode",
```
Comment (Aadarsh805) [inline data/DhanushNehru.json:33]: here as well

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**